cmd/cloudflared/tunnel/server.go

```diff
@@ -13,7 +13,11 @@ func runDNSProxyServer(c *cli.Context, dnsReadySignal, shutdownC chan struct{},
 	if port <= 0 || port > 65535 {
 		return errors.New("The 'proxy-dns-port' must be a valid port number in <1, 65535> range.")
 	}
-	listener, err := tunneldns.CreateListener(c.String("proxy-dns-address"), uint16(port), c.StringSlice("proxy-dns-upstream"), c.StringSlice("proxy-dns-bootstrap"), log)
+	maxUpstreamConnections := c.Int("proxy-dns-max-upstream-conns")
```

Comment:
Shouldn't this be `c.Int("max-upstream-conns")` to match the [definition](https://github.com/cloudflare/cloudflared/pull/290/files?diff=unified&w=1#diff-6f1cb455303a0da3ac60ed8b8cdcb845a5edb82939f56e2e05939cb625522a76)